Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG PYTHON_VERSION=3.10
+
+FROM python:${PYTHON_VERSION}-slim AS builder
+
+WORKDIR /app
+
+COPY . .
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=builder /app .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN python manage.py migrate
```

Comment:
Running database migrations during the build process might cause issues if the database is not ready when the container starts. Consider using an entrypoint script to handle database readiness checks and run migrations.